tests/conftest.py

```diff
@@ -176,7 +176,7 @@ def prepare_default_test_cases(self, param):
             @define_test_parameters(
                 suite_id=param[0], case_id=param[1], aux_list=param[2]
             )
-            class MyTestCase(test_case.BasicTest):
+            class MyTestCase(test_case.GreyTest):
```

Comment:
Can we run all the tests for Grey and Basic?

Reply:
Yes, it is the case now